logs-shipper/src/main/java/juice/modules/ProducerModule.java

```diff
@@ -0,0 +1,38 @@
+package juice.modules;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import config.ConfigFileFinder;
+import config.ConfigurationFactory;
+import config.ProducerConfiguration;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.util.Properties;
+
+public class ProducerModule extends AbstractModule {
+
+    private final String CONFIG_FILE_NAME = "producer.config";
+
+    @Provides
+    KafkaProducer<String, String> getKafkaProducer() {
```

Comment:
As the comment for the consumer binding - maybe initalize in the class that is using the producer?

Reply:
as agreed, we'll postpone it to the next bootcamp ;)